tests/test_server.py

```diff
@@ -290,6 +290,7 @@ def test_get_wal_full_path(self, tmpdir):
             "target_xid",
             "target_lsn",
             "expected_indices",
+            "target_immediate",
```

Comment:
I'd say to switch the positions of `expected_indices` and `target_immediate`. I mean, that seems a more logical way of reading the parametrized the tests ("given a set of conditions, I expected this")